### PR TITLE
Fix doctests

### LINKED ISSUE #90: Broken `pytest` configuration doesn't run doctest
**Describe the bug**

The current configuration of `pytest` in `setup.cfg` does not run `doctest` on the package modules. Although `--doctest-modules` is set, the `testpaths` setting only includes `tests`, so none of the module code is tested.

**To Reproduce**

If `testpaths = tests` is changed to `testpaths = tests pyrealm` then the doctests are discovered, many of which fail.

### PR DESCRIPTION
**Pull request by davidorme**
This closes #90 by:

* reimplementing the doctests by editing `setup.cfg` to look in `tests` and `pyrealm`.
* fixing all the accumulated broken doctests, which are mostly just changed arg names and array not scalar return types.

### CODE REVIEW

**Comment by davidorme** [comment]
Ack. I forked this branch off of an existing issue fix - not sure how I remove all those commits that are on #91

**Comment by MarionBWeinzierl** [comment]
> Ack. I forked this branch off of an existing issue fix - not sure how I remove all those commits that are on #91

I'd recommend a rebase (as described [here](https://stackoverflow.com/questions/36168839/how-to-remove-commits-from-a-pull-request)), but you can also do a revert (also described in that post).

**Review by MarionBWeinzierl — APPROVED** [review]
LGTM

File: pyrealm/hygro.py [hunk 1]
```diff
@@ -37,15 +37,15 @@ def calc_vp_sat(ta: NDArray, const: HygroConst = HygroConst()) -> NDArray:
...
         >>> temp = np.array([21])
-        >>> round(calc_vp_sat(temp), 6)
-        2.480904
+        >>> calc_vp_sat(temp).round(6)
+        array([2.480904])
         >>> from pyrealm.constants import HygroConst
         >>> allen = HygroConst(magnus_option='Allen1998')
-        >>> round(calc_vp_sat(temp, const=allen), 6)
-        2.487005
+        >>> calc_vp_sat(temp, const=allen).round(6)
```
Comment (a-smith-github) [inline pyrealm/hygro.py:44]: Example of where the help text misses the import. Although I suspect it's likely that the user could recover (user needs to understand the context from which they are accessing the method i.e. maybe they should use pyrealm.hygro.calc_vp_sat(args).

**Review by a-smith-github — COMMENTED** [review]
Looks really good David, just one suggestion about considering the import text in the examples (or at least one of them anyway).

**Pull request merged**